main/integration-tests/support/resources/projects/project.js

```diff
@@ -0,0 +1,38 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const _ = require('lodash');
+
+const Resource = require('../base/resource');
+
+class Project extends Resource {
+  constructor({ clientSession, id, parent }) {
+    super({
+      clientSession,
+      type: 'project',
+      id,
+      parent,
+    });
+
+    // In SWB, the user resource is mounted on two different namespaces: /api/users and /api/user
+    // The /api/user resource is meant to represent the current user.  This file represents the
+    // resource operations helper for /api/users. To represent the /api/user, see the current-user.js file
```

Comment:
I think this is copy and paste, you can remove this comment if you don't want to update it.  It is referencing /api/user instead of project